src/main/java/org/snomed/snowstorm/core/data/domain/jobs/ExportConfiguration.java

```diff
@@ -48,6 +48,15 @@ public class ExportConfiguration {
 	@Schema(description = "If refsetIds are included, this indicates that the export will be a refset-only export.")
 	private Set<String> refsetIds;
 
+	@Schema(defaultValue = "PENDING")
+	private String status;
+
+	@Schema
+	private String exportPath;
+
+	@Schema(defaultValue = "false")
+	private boolean startExport;
```

Comment:
@dmcgihtsdo As discussed before we shouldn't really need this workaround if we have made existing apis backward compatible. If this is a must then please make this transient as we don't want this to be persisted.